stylesheets/stylesheet.css

```diff
@@ -0,0 +1,70 @@
+.box {
+  height: 20em;
+  width: 15em;
+  border: 0.1em solid grey;
+  padding-left: 0.3em;
+}
+
+.container {
+  height: 400px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+img {
+  max-width: 100%;
+  margin: auto;
+}
+
+h2 {
+  color: green;
+}
+
+.checkout {
+  background-color: green;
+  color: white;
+  font-size: 115%

```

Comment:
I'm not a huge fan of percents for fonts, you'll want to use some sort of rem or em based for your fonts